### DISCUSSION TITLE
Ubuntu repos result in a 404?

### DISCUSSION THREAD

**Discussion by undeadindustries** [OpenVPN/openvpn3-linux#188] [Q&A - Our support arena] [answered]
When I run:
curl -fsSL https://swupdate.openvpn.net/community/openvpn3/repos/openvpn3-Ubuntu.list >/etc/apt/sources.list.d/openvpn3.list

I get: curl: (22) The requested URL returned error: 404

It doesn't matter if Ubuntu is lowercase or capital.

When I try to go to it in a browser I get:
Code: NoSuchKey
Message: The specified key does not exist.
Key: community/openvpn3/repos/openvpn3-debian.list
RequestId: FXVY46SMHPB8R64E
HostId: k+zc3EUPDP7u4fFA6tQJS++7ylTNURwB7DXquAi0deBAxKSHMFIH8noxdF39BWiC8wBNjSaw4/U

Did the URL in the docs change?

**Comment by dsommers** [comment, marked as answer]
"Ubuntu" is not a valid Ubuntu release name.  Check the table on [this page](https://community.openvpn.net/openvpn/wiki/OpenVPN3Linux).  So if you are running Ubuntu 22.04, the codename you need to use is `jammy`.
